Add description for new favorite button in Music Assistant (#39703)
home-assistant/home-assistant.github.io: source/_integrations/music_assistant.markdown

The Music Assistant integration creates media player entities for all players and groups available in MA, including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. This full functionality includes transfer of the playing queue of music from one player to another, automatic pausing of playback during announcements, and richer options for selecting the media for playback. These entities will display media information, playback progress, and playback controls.

### Favorite current song button

> The Music Assistant integration creates a button entity per player to favorite the current song. Pressing this button entity manually or by an automation will attempt to add the currently playing song to your Music Assistant favorites. This will work for local playing songs, tracks from streaming providers and if/when remote content such as Spotify Connect, AirPlay or even a radio station is playing, as long as the external source provides an artist and title combination (and optionally the album). Note that the button will be marked as unavailable if there is no content playable that could be favorited.

Reviewer: ```suggestion
The Music Assistant integration creates a button entity for each player to favorite the current song. Pressing this button (manually or by automation) adds the current song to your Music Assistant favorites. This works for local playing songs and tracks from streaming providers. It also works with remote content such as Spotify Connect, AirPlay, or a radio station, as long as the external source provides an artist and title combination (and optionally the album). Note that the button will be marked as unavailable if there is no content playable that could be favorited.
```
